packages/clima_data/test/models/weather_model_test.dart

```diff
@@ -56,7 +57,8 @@ void main() {
           humidity: 93,
           sunrise: 1611906189,
           sunset: 1611938637,
-          time: 0,
+          //  date: 1611919888,
```

Comment:
This comment looks unnecessary to me. Comments should generally explain _why_ a piece of code works that way, not _what_ it does.